pyimzml/ImzMLParser.py

```diff
@@ -293,6 +293,9 @@ def get_spectrum_as_string(self, index):
         intensity_string = self.m.read(lengths[1])
         return mz_string, intensity_string
 
+    def getionimage(self, mz_value, tol=0.1, z=None, reduce_func=sum):
+        getionimage(self, mz_value, tol, z, reduce_func)
+
```

Comment:
I'll remove it since this is your project. However I disagree with your logic. Many tools provide functionality both as an instance method and a standalone tool:

	>>> str.lower("ABC")
	'abc'
	>>> "ABC".lower()
	'abc'
	>>> from datetime import datetime
	>>> d = datetime.now()
	>>> datetime.isoformat(d)
	'2016-07-01T09:23:55.355108'
	>>> d.isoformat()
	'2016-07-01T09:23:55.355108'

Maybe the solution is to make new Imz class that uses the Parser, which has these tools in it.